.storybook/webpack.config.js

```diff
@@ -1,28 +1,18 @@
-/* eslint-disable no-underscore-dangle */
-/* eslint-disable no-param-reassign */
-const path = require('path');
-const dotenv = require('dotenv');
 const _ = require('underscore');
-const custom = require('../config/webpack/webpack.common')({
-    envFile: '../.env.production',
-});
-
-const env = dotenv.config({path: path.resolve(__dirname, '../.env.staging')}).parsed;
+const lodashMerge = require('lodash/merge');
+const getCommonConfig = require('../config/webpack/webpack.common');
 
 module.exports = ({config}) => {
-    config.resolve.alias = {
-        'react-native-config': 'react-web-config',
-        'react-native$': 'react-native-web',
-        '@react-native-community/netinfo': path.resolve(__dirname, '../__mocks__/@react-native-community/netinfo.js'),
-    };
+    const webConfig = getCommonConfig({envFile: '.env.production'});
+
+    lodashMerge(config.resolve, webConfig.resolve);
```

Comment:
Not sure it matters but for consistency should we use [`webpack-merge`](https://github.com/Expensify/App/blob/645f82e6491fa0995974e3d25546bf2e1658803f/config/webpack/webpack.dev.js#L2) ?

Reply:
Oh, nevermind this is only for `resolve` not the full config.